tools/c7n_azure/c7n_azure/filters.py

```diff
@@ -727,15 +727,21 @@ class CostFilter(ValueFilter):
     separately (e.g. SQL Server and SQL Server Databases). Warning message is logged if we detect
     different currencies.
 
-    Timeframe can be either number of days before today or one of:
+    Timeframe options:
 
-    WeekToDate,
-    MonthToDate,
-    YearToDate,
-    TheLastWeek,
-    TheLastMonth,
-    TheLastYear
+      - Number of days before today
 
+      - All days in current period until today:
```

Comment:
I think it would be clearer if we change "current period" to "current calendar period". Same for line 740

Happy to discuss this further